www/js/control/ProfileSettings.jsx

```diff
@@ -150,54 +255,206 @@ const ProfileSettings = () => {
           So we don't need to call refreshCollectSettings here. */
     }
 
-    const toggleLowAccuracy = function() {
+
+    const safeToggle = function() {
+        if(toggleTime){
+            const prevTime = toggleTime.getTime();
```

Comment:
if I followed the thread correctly, this is a workaround until the plugin helpers are also migrated to react
https://github.com/e-mission/e-mission-phone/pull/1009#issuecomment-1707002882
https://github.com/e-mission/e-mission-phone/pull/1009#issuecomment-1707011173

Reply:
Correct that it is a workaround to prevent the function from being called twice in a row and causing the "bouncing" behavior that was our concern with the switch. When we migrate plugin helpers we should revisit this to see if it can be resolved.